renovate/automerge-all-package-version-minor-bump.json

```
@@ -0,0 +1,25 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "ignorePaths": [
+        "**/node_modules/**",
+        "**/bower_components/**",
+        "**/vendor/**",
+        "**/examples/**",
+        "**/__tests__/**",
+        "**/test/**",
+        "**/tests/**",
+        "**/__fixtures__/**"
+    ],
+    "printConfig": true,
+    "packageRules": [
+        {
+            "matchUpdateTypes": [
+                "minor"
+            ],
+            "matchCurrentVersion": "!/^(0|v0)/",
```

Does renovate support negative regexes?

Alternatively:
```suggestion
        {
            "matchUpdateTypes": [
                "minor"
            ],
            "matchCurrentVersion": "/^v?[1-9]/",
```

---

It does. We are using it at New Relic: https://github.com/newrelic/coreint-automation/blob/af3a745b9dffa3c74f916d394e21b463615a5b17/renovate-base.json5#L35

It works :)